sneak_peek.md

```diff
@@ -17,21 +17,25 @@ This is essentially a placeholder for the next release note ...
     fixed-size variables followed by record variables.
 
 * New Limitations
-  + ADIOS driver are ready only. No vard, varn, low-level, and nonblocking support.
+  + ADIOS driver is ready only. There is also no vard, varn and low-level API support.
 
 * Update configure options
   + Enable ADIOS support.
-    - `--enable-adios`: enable NetCDF4 format classic mode support
-    - `--with-adios=/path/to/netcdf-4`: path to NetCDF-4 library installation
+    - `--enable-adios`: enable reading BP formated file
+    - `--with-adios=/path/to/netcdf-4`: path to ADIOS library installation
   + When building with NetCDF-4 feature, using NetCDF-4 library built with
     PnetCDF enabled, i.e. --enable-pnetcdf, is not supported. See
     [Issue #33](https://github.com/Parallel-NetCDF/PnetCDF/issues/33).
 
-* Update configure options
-  + none
-
 * New constants
-  + none
+  + NC_BP = 65536
```

Comment:
No need to mention the values of new constants.
See RELEASE_NOTES for examples of introducing new constants.